frontend/src/pages/application/Pipelines.vue

```diff
@@ -50,7 +50,7 @@
             @stage-deleted="(stageIndex) => stageDeleted(pipeline, stageIndex)"
         />
     </div>
-    <EmptyState v-else :featureUnavailable="!featureEnabled" :featureUnavailableMessage="'This feature is not available'">
+    <EmptyState :featureUnavailable="true">
```

Comment:
@Pezmc why have you sset this always `true`?

Reply:
Because I pushed my tests while rushing to be done 🙈